src/mon/mon_types.h

```diff
@@ -38,22 +38,6 @@ enum paxos_svc_type : int {
   PAXOS_NUM
 };
 
-
```

Comment:
might want to change the prefix of the title to "mon/mon_types.h".

Reply:
Done